menu/main_menu.sh

```diff
@@ -25,33 +25,21 @@ choice=$(whiptail --title "Main Menu" --radiolist "Choose what you want to do.\n
 
 case "$choice" in
     "Additional Apps")
-        if network_ok
-        then
-            run_script MENU additional_apps
-        fi
+        run_script MENU additional_apps
     ;;
     "Nextcloud Configuration")
-        if network_ok
-        then
-            run_script MENU nextcloud_configuration
-        fi
+        run_script MENU nextcloud_configuration
     ;;
     "Server Configuration")
-        if network_ok
-        then
-            run_script MENU server_configuration
-        fi
+        run_script MENU server_configuration
     ;;
     "Update Nextcloud")
         if [ -f $SCRIPTS/update.sh ]
         then
             bash $SCRIPTS/update.sh
         else
-            if network_ok
-            then
-                download_script STATIC update
-                bash $SCRIPTS/update.sh
-            fi
+            download_script STATIC update
+            bash $SCRIPTS/update.sh
```

Comment:
Just if you wonder: I've made that change because to get into this script we will have already done a network check with should be enough, imho...

Reply:
IMHO, we don't need the network checks at all.

Reply:
I removed them 👍